README.md

```diff
@@ -1 +1,109 @@
-# angular-concepts-readme
+# Angular Core Concepts
+
+## Overview
+
+In this lesson, we are going to look at the core concepts that we will use in every Angular application. Some will be familiar to you, however some may seem a bit strange. Don't worry if they take time to sink in!
+
+## Objectives
+
+- Define MVC (Model-View-Controller), MVVM (Model-View-ViewModel)
+- Describe one-way and two-way data-binding
+- Describe Angular's two-way binding
+- Describe Dependency Injection
+- Describe a Single Page Application and common features
+- Demonstrate server-side rendering concepts
+
+## Single Page Applications
+
+Angular allows us to create single page applications (SPAs). The means that the user goes to one page and very rarely will have to refresh/leave that page to go to other parts of the website.
+
+Single page applications handle all the routing, calling different parts of your codebase dependant on what URL the user is currently at.
```

Comment:
Can you flesh this out more, maybe with an example of a real application? Doesn't have to be technical. As you discuss more about how the routing works, this example may also be a good transition into MVC/MVVM - move it below the description of traditional rendering below